packages/sdk/src/integration/logger.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+import util from 'util'
+
+const serializeForBotMessage = (args: Parameters<typeof util.format>) => {
+  if (process.env['BP_LOG_FORMAT'] === 'json') {
+    return JSON.stringify({ msg: util.format(...args), visible_to_bot_owner: true })
+  } else {
+    const [format, ...param] = args
+    return util.format(`[For Bot Owner] ${format}`, ...param)
+  }
+}
+
+export const integrationLogger = {
```

Comment:
**chore:** as discussed, this object should also expose `info`, `warn`, etc... for logs that are not meant to be seen by the bot